dspace-api/src/main/java/org/dspace/content/RelationshipServiceImpl.java

```diff
@@ -52,45 +58,103 @@ public Relationship create(Context context) throws SQLException, AuthorizeExcept
     @Override
     public Relationship create(Context context, Relationship relationship) throws SQLException, AuthorizeException {
         if (isRelationshipValidToCreate(context, relationship)) {
-            if (!authorizeService.isAdmin(context)) {
+            if (authorizeService.authorizeActionBoolean(context, relationship.getLeftItem(), Constants.WRITE) ||
+                authorizeService.authorizeActionBoolean(context, relationship.getRightItem(), Constants.WRITE)) {
+                updatePlaceInRelationship(context, relationship, true);
+                return relationshipDAO.create(context, relationship);
+            } else {
                 throw new AuthorizeException(
-                    "Only administrators can modify relationship");
+                    "You do not have write rights on this relationship's items");
             }
-            updatePlaceInRelationship(context, relationship);
 
-            return relationshipDAO.create(context, relationship);
         } else {
             throw new IllegalArgumentException("The relationship given was not valid");
         }
     }
 
-    private void updatePlaceInRelationship(Context context, Relationship relationship) throws SQLException {
+    @Override
+    public void updatePlaceInRelationship(Context context, Relationship relationship, boolean isCreation)
+        throws SQLException, AuthorizeException {
+        Item leftItem = relationship.getLeftItem();
         List<Relationship> leftRelationships = findByItemAndRelationshipType(context,
-                                                                             relationship.getLeftItem(),
+                                                                             leftItem,
                                                                              relationship.getRelationshipType(), true);
+        Item rightItem = relationship.getRightItem();
         List<Relationship> rightRelationships = findByItemAndRelationshipType(context,
-                                                                              relationship.getRightItem(),
+                                                                              rightItem,
                                                                               relationship.getRelationshipType(),
                                                                               false);
 
-        if (!leftRelationships.isEmpty()) {
-            leftRelationships.sort((o1, o2) -> o2.getLeftPlace() - o1.getLeftPlace());
-            for (int i = 0; i < leftRelationships.size(); i++) {
-                leftRelationships.get(leftRelationships.size() - 1 - i).setLeftPlace(i + 1);
+        context.turnOffAuthorisationSystem();
```

Comment:
Why are we turning off authorization here?  I'm a little nervous anytime I see us turn off authorization *without* an inline comment to describe it.  Shouldn't the user have proper permissions to do these updates?